### PR TITLE
feat: added new logged in Navbar and WalletPopup UI components (#295)

### LINKED ISSUE #295: [System Contract Dapp] wallet dropdown
### Problem

System Contract Dapp needs a wallet dropdown

### Solution

Add a wallet dropdown

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**: This PR added new UI component for the logged in Navbar and WalletPopup along with the necessary APIs for the functionalities

The WalletPopup can:

- dynamically displays user information (EVM Address, accountID,, and balance)
     + querying EVM address from Metamask
     + querying accountId from mirror-node using the EVM address
     + querying balance through relay RPC-API ("eth_getBalance")
  
- navigates user to the correct Hashscan page when the `Explore Account` button is clicked

- disconnects users by removing all account info related cookies and navigate user back to landing page

** UI Update **:
a. New logged in Navbar UI
image

b. WalletPopup UI
image

Fixes #295 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: system-contract-dapp-playground/src/api/mirror-node/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+/**
+ * @dev get Hedera native account ID from EVM address
+ *
+ * @params evmAddress string
+ *
+ * @returns string
+ */
+export const getAcocuntIdFromEvmAddress = async (evmAddress: string): Promise<MirrorNodeResult> => {
+  try {
+    const res = await fetch(`${HEDERA_NETWORKS.testnet.mirrorNodeUrl}/accounts/${evmAddress}`);
```
Comment (Nana-EC) [inline system-contract-dapp-playground/src/api/mirror-node/index.ts:33]: Network should be dynamic. It won't always be testnet.
I'd recommend passing the network value in

**Review by Nana-EC — CHANGES_REQUESTED** [review]
1st pass.
I think the account get info needs the current environment variable.

Additionally I think it's tike we started adding some basic tests to ensure no regressions.

You'll want to focus on the backend info.
Example is the correct mirror url called or provided when a network environment is set

File: system-contract-dapp-playground/src/components/navbar/index.tsx [hunk 2]
```diff
@@ -23,29 +23,146 @@
...
+      const accountsInfo = loadAccountInfoFromCookies();
+
+      // just best pratice to handle error as this would rarely happen as we have already gated all the checks in middleware.js
+      if (
+        !accountsInfo.isConnected ||
+        !accountsInfo.accounts ||
+        !accountsInfo.network ||
+        accountsInfo.error
+      ) {
+        CommonErrorToast({
```
Comment (Nana-EC) [inline system-contract-dapp-playground/src/components/navbar/index.tsx:59]: Q: does this log anything?
IF not should be be logging `accountInfo` to help troubleshoot

File: system-contract-dapp-playground/src/utils/constants.ts [hunk 3]
```diff
@@ -100,5 +103,22 @@ export const HEDERA_NETWORKS = {
       decimals: 18,
     },
     blockExplorerUrls: 'http://localhost:8080',
+    mirrorNodeUrl: '127.0.0.1:5600/api/v1',
```
Comment (Nana-EC) [inline system-contract-dapp-playground/src/utils/constants.ts:106]: Q: Does this need the `http://` prefix?

File: system-contract-dapp-playground/src/utils/constants.ts [hunk 4]
```diff
@@ -100,5 +103,22 @@ export const HEDERA_NETWORKS = {
...
+  '/hts-hip-206',
+  '/hrc-719',
+  '/exchange-rate-hip-206',
+  '/prng-hip-351',
+  '/erc-20',
+  '/erc-721',
+];
+
+/** @notice hashcan baseURL */
+export const HASHSCAN_BASE_URL = 'https://hashscan.io';
```
Comment (Nana-EC) [inline system-contract-dapp-playground/src/utils/constants.ts:124]: Maybe you can add the hashscan url to the `HEDERA_NETWORKS` object.
If not create an object here or a method in helpers to handle hashscan for each env i.e. `http://127.0.0.1:8080`, `https://hashscan.io/previewnet/dashboard`, `https://hashscan.io/testnet/dashboard` and `https://hashscan.io/mainnet/dashboard`

**Review by Nana-EC — COMMENTED** [review]
LG. 2 nits

File: system-contract-dapp-playground/src/components/navbar/index.tsx (hunk 2 above)
Comment (quiet-node) [reply system-contract-dapp-playground/src/components/navbar/index.tsx:59]: yes it does log the error in the client-side console. I should change the message from "Unknown error appeared..." to "'See client console for more information'" though.

File: system-contract-dapp-playground/src/utils/constants.ts (hunk 4 above)
Comment (quiet-node) [reply system-contract-dapp-playground/src/utils/constants.ts:124]: Oh yeah I did add those urls to the `HEDERA_NETWORKS` they are under `blockExplorerUrl`. However, the getAccountIdFromEvmAddress() API require the `Hashscan` url to have a different param like `https://hashscan.io/testnet/accounts`. And in the future we will need different params for different purposes like `transactions` or `contracts`, etc. so I think a `HASHSCAN_BASE_URL` is necessary.

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**